package.json

```diff
@@ -424,6 +424,14 @@
           "when": "view == objectExplorer && config.mssql.enableRichExperiences && viewItem =~ /\\bfilterable=true\\b.*\\bhasFilters=true\\b/",
           "group": "inline@2"
         },
+        {
+          "command": "mssql.schemaCompare",
+          "when": "view == objectExplorer && config.mssql.enableRichExperiences && viewItem =~ /\\btype=(disconnectedServer|Server|Database)\\b/"
```

Comment:
can you make sure if this works in disconnected databases connections?

Reply:
Disconnected DB's will have the Schema Compare option in the context menu. ![image](https://github.com/user-attachments/assets/30174ae5-0e77-4cde-a83e-dfb5f659097b)

Clicking on it will open schema compare.
![image](https://github.com/user-attachments/assets/85f2416f-bf71-49e6-9f9b-30b90fa6d3b0)

Which is the same way ADS opens up Schema Compare with disconnected connections too:
![image](https://github.com/user-attachments/assets/2751b003-c15a-4699-a451-6a89ce9016b5)